roles/kubernetes-apps/ansible/templates/coredns-deployment.yml.j2

```diff
@@ -30,8 +30,10 @@ spec:
         beta.kubernetes.io/os: linux
       serviceAccountName: coredns
       tolerations:
-        - key: node-role.kubernetes.io/master
```

Comment:
This toleration is very valuable. I don't think we want to allow scheduling on any node that has just NoSchedule. We only want it for master taint...